tests/windows_test.go

```diff
@@ -93,15 +86,13 @@ var _ = Describe("[Serial][sig-compute]Windows VirtualMachineInstance", Serial,
 			BeforeEach(func() {
 				By("Starting the windows VirtualMachineInstance")
 				var err error
-				windowsVMI, err = virtClient.VirtualMachineInstance(testsuite.NamespaceTestDefault).Create(context.Background(), windowsVMI, metav1.CreateOptions{})
+				windowsVMI, err = virtClient.VirtualMachineInstance(testsuite.NamespaceTestDefault).Create(context.Background(), newWindows(), metav1.CreateOptions{})
 				Expect(err).ToNot(HaveOccurred())
 				libwait.WaitForSuccessfulVMIStart(windowsVMI)
-
-				cli = winrnLoginCommand(windowsVMI)
 			})
 
 			It("[test_id:240]should have correct UUID", func() {
-				command := append(cli, "wmic csproduct get \"UUID\"")
+				command := append(winrnLoginCommand(windowsVMI), "wmic csproduct get \"UUID\"")
```

Comment:
```suggestion
				command := append(winrmLoginCommand(windowsVMI), "wmic csproduct get \"UUID\"")
```

typo: `winrn` != `winrm`

Reply:
Done, good catch